<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>灰度图像增强</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>461</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>分段线性变换</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>幂变换</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>分界点a</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="value_linear_a">
           <property name="maximum">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>分界点b</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="value_linear_b">
           <property name="maximum">
            <number>256</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>灰度拉升系数a</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="value_power_a">
           <property name="maximum">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>gamma系数b</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="value_power_b">
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="submit_linear">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="submit_power">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
